### PR TITLE
Contribution guide

### PR DESCRIPTION
**Pull request by myfunnyandy**

### CODE REVIEW

File: contributing-guide/translations/index.njk [hunk 1]
```diff
@@ -4,4 +4,45 @@ title: 02· Translations
...
 <h1 id="translations">Translations</h1>
 
-<p class="main-paragraph">TBD</p>
+<p class="main-paragraph">Ways to contribute translation Penpot</p>
+
+<h3 id="translations-howto">How to become a Penpot translator</h3>
+<p>We are using <a href="https://weblate.org/" target="_blank">Weblate</a> as translation platform, so the first thing you need to be a Penpot translator is to have a Weblate account (you can <a href="https://hosted.weblate.org/accounts/register/" target="_blank">register here</a>).</p>
+<p>To start translating at Penpot:</p>
+<ol>
+<li>Open a <a href="<a href="https://hosted.weblate.org/accounts/register/" target="_blank">github issue</a> giving details about the language you want to translate (language), the type of translation (new language, new translation or change an existing translation) and your Weblate user.</li>
```
Comment (hirunatan) [inline contributing-guide/translations/index.njk:13]: broken link

File: contributing-guide/translations/index.njk [hunk 2]
```diff
@@ -4,21 +4,28 @@ title: 02· Translations
...
-<p class="main-paragraph">Ways to contribute translation Penpot</p>
+<p class="main-paragraph">Thank you for interest in contribute translating Penpot. Here you will find ways to do it.</p>
 
 <h3 id="translations-howto">How to become a Penpot translator</h3>
 <p>We are using <a href="https://weblate.org/" target="_blank">Weblate</a> as translation platform, so the first thing you need to be a Penpot translator is to have a Weblate account (you can <a href="https://hosted.weblate.org/accounts/register/" target="_blank">register here</a>).</p>
 <p>To start translating at Penpot:</p>
 <ol>
 <li>Open a <a href="<a href="https://hosted.weblate.org/accounts/register/" target="_blank">github issue</a> giving details about the language you want to translate (language), the type of translation (new language, new translation or change an existing translation) and your Weblate user.</li>
 <li>If everything is correct we will get back to you providing you permissions to the actions needed.</li>
+<li>You also might want to take a look at the guide for <a href="<a href="https://docs.weblate.org/en/latest/user/translating.html" target="_blank">Translating using Weblate</a>.</li>
```
Comment (hirunatan) [inline contributing-guide/translations/index.njk:15]: broken link

File: contributing-guide/translations/index.njk [hunk 3]
```diff
@@ -4,21 +4,28 @@ title: 02· Translations
...
 </ol>
 
 <h3 id="translations-howto">Add a new language</h3>
-<p>mmm</p>
-1- entra en https://hosted.weblate.org/projects/penpot/frontend/es/
-2- pulsa "comenzar una "
-3- añade traducciones
+<p>To add a language that is still not among the Penpot language options:</p>
+<ol>
+<li>Go to the <a href="<a href="https://hosted.weblate.org/projects/penpot/frontend/" target="_blank">languages list</a>.</li>
```
Comment (hirunatan) [inline contributing-guide/translations/index.njk:21]: broken link

File: contributing-guide/translations/index.njk [hunk 4]
```diff
@@ -28,10 +35,12 @@ title: 02· Translations
 <li>Click the edit button (pencil icon) close to the name of the language where you want to add the missing translation or translations.</li>
```
Comment (hirunatan) [inline contributing-guide/translations/index.njk:35]: broken link (in previous line)

File: contributing-guide/code-contributions/index.njk [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+<p>Go to the <a href="/contributing-guide/translations">Technical guide</a> to get detailed explanations about how to get Penpot application and run it locally, to test it or make changes to it.</p>
+
+<h3 id="code-contributions-pull-requests">Pull requests</h3>
+<p>If you want propose a change or bug fix with the Pull-Request system firstly you should carefully read the DCO section and format your commits accordingly.</p>
+<p>If you intend to fix a bug it's fine to submit a pull request right away but we still recommend to file an issue detailing what you're fixing. This is helpful in case we don't accept that specific fix but want to keep track of the issue.</p>
+<p>If you want to implement or start working in a new feature, please open a <b>question</b> / <b>discussion</b> issue for it. No pull-request will be accepted without previous chat about the changes, independently if it is a new feature, already planned feature or small quick win.</p>
+<p>If is going to be your first pull request, You can learn how from <a href="https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github" target="_blank">this free video series</a>.</p>
```
[the remark this answers is not in the document]
Comment (myfunnyandy) [reply contributing-guide/code-contributions/index.njk:16]: Link fixed. It already has target blank.

File: contributing-guide/code-contributions/index.njk [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+<h1 id="code-contributions">Core code contributions</h1>
+
+<p class="main-paragraph">Details to know how to improve Penpot's core code</p>
+
+<h3 id="code-contributions-techguide">Technical guide</h3>
+<p>Go to the <a href="/contributing-guide/translations">Technical guide</a> to get detailed explanations about how to get Penpot application and run it locally, to test it or make changes to it.</p>
+
+<h3 id="code-contributions-pull-requests">Pull requests</h3>
+<p>If you want propose a change or bug fix with the Pull-Request system firstly you should carefully read the DCO section and format your commits accordingly.</p>
```
[the remark this answers is not in the document]
Comment (myfunnyandy) [reply contributing-guide/code-contributions/index.njk:13]: good one, done

**Pull request merged**